pkg/loader/compose/compose.go

```diff
@@ -398,3 +402,10 @@ func handleServiceType(ServiceType string) (string, error) {
 func normalizeServiceNames(svcName string) string {
 	return strings.Replace(svcName, "_", "-", -1)
 }
+
+func checkLabelsPorts(noOfPort int, labels string, svcName string) error {
+	if noOfPort == 0 && labels == "NodePort" || labels == "LoadBalancer" {
+		return errors.Errorf("%s defined in service %s with no ports present. Issues may occur when bringing up artifacts.", labels, svcName)
```

Comment:
@containscafeine , I have variable in my error message. so I used `errors.Errorf()` instead of `errors.New()`